ts/api/api.ts

```diff
@@ -123,6 +124,11 @@ export interface OpenYoloApi {
    * Cancels the last pending OpenYOLO request.
    */
   cancelLastOperation(): Promise<void>;
+
+  /**
+   * Sets the desired log level for the OpenYOLO library.
+   */
+  setLogLevel(level: LogLevel): Promise<void>;
```

Comment:
I meant that the provider could use the `LogLevel` to do something appropriate. We don't expose helpers for it. 
I can change to the `OnDemandOpenYoloApi`, which makes most of the changes to the api unnecessary.